deploy/gateway/templates/NOTES.txt

```diff
@@ -15,7 +15,7 @@ Admin Console URL - {{ printf "https://%s.%s" .Values.twingate.network (include
 
 1. The Gateway is accessible via its LoadBalancer IP. It may take a few minutes for the IP address to be available.
 
-Watch the IP address with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ include "gateway.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}''
+Watch the IP address with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ include "gateway.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0]}''
```

Comment:
Should we change this to `Watch the Load Balancer address with ...`?

Reply:
Yes! Updated in [26d1cb0](https://github.com/Twingate/kubernetes-access-gateway/pull/30/commits/26d1cb09c648eed7ab4cf749f118daf42b447daf)